Code/src/nlp/native_version.py

```diff
@@ -0,0 +1,33 @@
+import codecs
+import pandas as pd
+import pytrie
+import pickle
+
+
+def load_data_native(path_to_data):
+    with codecs.open(path_to_data, "r", encoding='utf-8', errors='ignore') as fdata:
+        grams = pd.read_table(fdata, names=["freq", "first", "second"])
+
+    grams['freq'] = grams['freq'].apply(lambda x: (x,))
+    freqs = grams['freq'].values
+    phrases = grams['first'] + " " + grams['second']
+    res = dict(zip(phrases, freqs))
+    pytrie1 = pytrie.StringTrie(res)
+    with open('pytrie.pkl', 'wb') as output:
+        pickle.dump(pytrie1, output, pickle.HIGHEST_PROTOCOL)
+    return pytrie1
+
+
+def otherAuto(trie1, word):
```

Comment:
some documentation would be nice; hard to understand what this method does

also add comments in general :P